components/JsonLd/JsonLd.tsx

```
@@ -0,0 +1,30 @@
+}
+
+export function JsonLd({ data }: JsonLdProps) {
+  // JSON.stringify escapes characters that could break out of the script tag
+  // This is safe for trusted data from our database
+  const jsonString = JSON.stringify(data);
+
+  return (
+    <script
+      type="application/ld+json"
+      // Safe: JSON.stringify escapes special chars, data is from trusted DB
+      dangerouslySetInnerHTML={{ __html: jsonString }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)